tests/css_color/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -15,6 +49,27 @@ exports[`hexcolor.css 1`] = `
   -webkit-color: #873;
   -moz-color: #6bcdef;
   -ms-color: #aabbcc;
+  color: #F09F;
```

Comment:
Bug in `postcss-values-parser`, PR - https://github.com/shellscape/postcss-values-parser/pull/44. After merge i will update version and snapshots.